srtcore/core.h

```diff
@@ -934,7 +934,7 @@ class CUDT
     int32_t m_iAckSeqNo;                         // Last ACK sequence number
     sync::atomic<int32_t> m_iRcvCurrSeqNo;       // (RCV) Largest received sequence number. RcvQTh, TSBPDTh.
     int32_t m_iRcvCurrPhySeqNo;                  // Same as m_iRcvCurrSeqNo, but physical only (disregarding a filter)
-
+    bool m_bufferWasFull;                        // Indicate that RX buffer was full last time a ack was sent
```

Comment:
With respect to the naming convention the `b` should be added as a prefix indicating the `boolean` type.
```suggestion
    bool m_bBufferWasFull;                        // Indicate that RX buffer was full last time an ACK was sent.
```